packages/libraries/main/src/matcher/dictionary/variants/matching/l33t.ts

```diff
@@ -27,7 +27,11 @@ class MatchL33t {
     const enumeratedSubs = this.enumerateL33tSubs(
       this.relevantL33tSubtable(password, zxcvbnOptions.l33tTable),
     )
-    for (let i = 0; i < enumeratedSubs.length; i += 1) {
+    const length =
+      enumeratedSubs.length <= zxcvbnOptions.l33tMaxSubstitutions
+        ? enumeratedSubs.length
+        : zxcvbnOptions.l33tMaxSubstitutions
```

Comment:
Maybe use `Math.min()` here?

```suggestion
    const length = Math.min(enumeratedSubs.length, l33tMaxSubstitutions)
```

Reply:
This is kind of embarrassing. I know there was an easier way but could get it into my head. :D